<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>TRIANGLE</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>30</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>RHOMBUS</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>70</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Click Here To Show...</string>
    </property>
    <property name="toolTipDuration">
     <number>2</number>
    </property>
    <property name="statusTip">
     <string>Click Here To Show...</string>
    </property>
    <property name="text">
     <string>SHOW</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>180</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>TRANSLATE</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>220</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SCALE</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>260</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ROTATE</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>300</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Click to proceed..</string>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="statusTip">
     <string>Click to proceed..</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>178</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Choice</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>110</y>
      <width>248</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>380</y>
      <width>88</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter X Factor</string>
    </property>
    <property name="toolTipDuration">
     <number>4</number>
    </property>
    <property name="statusTip">
     <string>Enter X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>350</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>380</y>
      <width>88</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter Y Factor</string>
    </property>
    <property name="toolTipDuration">
     <number>3</number>
    </property>
    <property name="statusTip">
     <string>Enter Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>450</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Click to proceed..</string>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="statusTip">
     <string>Click to proceed..</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1286</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
